capitalize_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Decorator'
```

Comment:
- [ ] Please consider fixing the casing in your `require_relative`. The name of the file you're importing is `decorator`, not `Decorator`. 🤓 🔧